scss/tech.scss

```diff
@@ -1,44 +1,59 @@
+.all-cards {
+  display: flex;
+  flex-wrap: wrap;
+  margin-bottom: 32px;
+  padding: 40px 40px 40px 100px;
+}
+
 .card {
   align-items: center;
   background-color: $green;
   border-radius: 4px;
   display: flex;
   flex-direction: column;
+  margin-bottom: 42px;
+  margin-right: 42px;
   max-width: 100%;
   padding: 16px;
   position: relative;
-  width: 200px;
+  width: 215px;
 
   &::before {
     border: 2px solid $green;
     border-radius: 4px;
     content: '';
     height: 100%;
-    left: -16px;
+    left: -14px;
     position: absolute;
-    top: -16px;
+    top: -14px;
     width: 100%;
     z-index: -1;
   }
+
+  &:hover::before {
+    transform: translate(22px, 22px);
+    transition-duration: 1s;
```

Comment:
This property should go in the definition of the class i.e in `&::before` and change for: `transition: transform 1s;`